Desktop/components/JASP/Widgets/LabelEditorWindow.qml

```diff
@@ -135,7 +135,7 @@ FocusScope
 						{
 							anchors.fill:		selectionRectangle
 							acceptedButtons:	Qt.LeftButton
-							cursorShape:		Qt.PointingHandCursor
+							cursorShape:		Qt.CrossCursor	
```

Comment:
I've always wanted to have a consistent mouse shape on the cell pointer in all table views.